flake8_picky_parentheses/_brackets_position.py

```diff
@@ -31,10 +31,16 @@ def first_in_line(self, cords):
 
     def last_in_line(self, cords):
         line = self.source_code_lines[cords[0] - 1]
-        return all(
-            line[col] in (" ", "\t", "\n")
-            for col in range(cords[1] + 1, len(line))
-        )
+        for i in range(cords[1] + 1, len(line)):
+            if line[i] == " ":
+                continue
+            elif line[i] == "#":
+                return True
+            else:
+                return all(
+                    line[col] in (" ", "\t", "\n")
+                    for col in range(cords[1] + 1, len(line))
+                )
```

Comment:
Think about what this means.

1. go through all characters starting at `cords`
   1. If it's a space, keep looking
   2. If it's a `#`, ignore the rest of the line
   3. Else, return whether the rest of the line only consists of spaces, tabs, and new lines.
  
  What about this example?
  ```python
  a = ( \t  # some comment
      1, 2)
```

Reply:
An alternative idea I just had: why don't we use the tokens in that line? I don't really like the idea of hardcoding what constitutes whitespace and what a comment in Python.